### ISSUE TITLE
No Cache for ios 

### ISSUE THREAD

**Issue by ytaiff** [zhangtaii/react-native-advanced-webview#6]
I use it but when offline ,it's no use noCache

**Comment by bsutton** [comment]
We are seeing the same problem.

**Comment by mattryles** [comment]
Yeah its come out of the back of the NSURLProtocol not longer supporting the HTTP protocol, only non HTTP and HTTPS protocols :(

**Comment by NishidaRyu416** [comment]
Me either, this module does not work on iOS.
